### ISSUE TITLE
 can't download Great kart template

### ISSUE THREAD

**Issue by krishdeva01** [dev-rathankumar/greatkart-course#1]
Can you please send me a great kart template folder...I can't able to download it from the course..

**Comment by maulik1626** [comment]
yes the template you gave is different from what you're working on...

**Comment by Asadbek-Web** [comment]
Mr.Kumar can you send greatcart practice templates, i can't download from your website
